contracts/token/ERC20/extensions/ERC20Votes.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-// OpenZeppelin Contracts v4.4.0 (token/ERC20/extensions/ERC20Votes.sol)
+// OpenZeppelin Contracts v4.3.2 (token/ERC20/extensions/ERC20Votes.sol)
```

Comment:
This should not be touched.

The scripts will overwrite this when we release anyway, but it would be better not to change it.